source/IrcDotNet/StandardIrcClient.cs

```diff
@@ -162,20 +160,30 @@ public void Connect(IPAddress address, bool useSsl, IrcRegistrationInfo registra
             if (registrationInfo == null)
                 throw new ArgumentNullException("registrationInfo");
 
-            Connect(address, DefaultPort, useSsl, registrationInfo);
+            Connect(new IPEndPoint(address, DefaultPort), useSsl, registrationInfo);
         }
 
-        /// <inheritdoc cref="Connect(EndPoint, bool, IrcRegistrationInfo)" />
+        /// <inheritdoc cref="Connect(IPEndPoint, bool, IrcRegistrationInfo)" />
         /// <param name="address">An IP addresses that designates the remote host.</param>
         /// <param name="port">The port number of the remote host.</param>
-        public void Connect(IPAddress address, int port, bool useSsl, IrcRegistrationInfo registrationInfo)
+        public void Connect(IPEndPoint address, int port, bool useSsl, IrcRegistrationInfo registrationInfo)
```

Comment:
I don't see the reason for this change at all. It's a bit odd, considering an `IPEndPoint` contains a port too (and there is already a method overload for `IPEndPoint`).